app/controllers/api/v3/public/card_faces_controller.rb

```diff
@@ -0,0 +1,8 @@
+module API
+  module V3
+    module Public
+      class Api::V3::Public::CardFacesController < JSONAPI::ResourceController
```

Comment:
As I asked in Discord, do we really want to expose these as their own objects/entities?  Same question goes for printing faces.